src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/AsnCharacterStringEncodings.cs

```diff
@@ -377,10 +421,10 @@ public override int GetMaxCharCount(int byteCount)
     /// Compatibility encoding for T61Strings. Interprets the characters as UTF-8 or
     /// ISO-8859-1 as a fallback.
     /// </summary>
-    internal class T61Encoding : Text.Encoding
+    internal sealed class T61Encoding : Encoding
```

Comment:
All of the ones that read character data are guarded.  Anything that's an invalid UTF-8 sequence falls back to Latin-1 for compat with Windows' T61 implementation.

From character data to bytes we require properly formed surrogates and we output UTF-8, so no fallbacks are needed.  GetMaxCharCount reports Latin-1's number, since it's the more conservative of the two.  No fallback needed since there's no data to read.